articles/2021/11/sustainable-applications-using-azure-cdn.md

```diff
@@ -0,0 +1,259 @@
+
+```hcl
+
+resource "azurerm_cdn_profile" "cdnGreenApp" {
+  name                = "cdn-profile-app-green-dev-001"
+  location            = azurerm_resource_group.rg_green_front_fr_001.location
+  resource_group_name = azurerm_resource_group.rg_green_front_fr_001.name
+  sku                 = "Standard_Verizon"
+}
+
+resource "azurerm_cdn_endpoint" "cdnGreenAppEndpoint" {
+  depends_on = [
+    azurerm_app_service.greenAppService
+  ]
+  name                = "cdn-endpoint-app-green-dev-001"
+  profile_name        = azurerm_cdn_profile.cdnGreenApp.name
+  location            = azurerm_resource_group.rg_green_front_fr_001.location
+  resource_group_name = azurerm_resource_group.rg_green_front_fr_001.name
+  optimization_type   = "GeneralWebDelivery"
+  origin_host_header  = "${azurerm_app_service.greenAppService.name}.azurewebsites.net"
+
+  origin {
+    http_port  = "80"
+    https_port = "443"
+    name       = "greenAppFrontend"
+    host_name  = "${azurerm_app_service.greenAppService.name}.azurewebsites.net"
+  }
+}
+
+
+```
+
+### Provision an Azure CDN from the web portal
+
+For configuring a CDN endpoint for DSA optimization by using the Azure portal:
+
+1. **From the created Web App, select Networking on the Settings section:**
+
+![AzureNetworkingWebApp](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/rlgvx7en7nlz7dzrdtdb.png)
+
+1. **Select Azure CDN from More networking feature**
+2. **Register the new Endpoint and CDN profile with Standard Verizon pricing tier**
+
+![AzureCDNProfile](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/xgwgddsy8cchbccbz87v.png)
+
+
+3. **Once created, the new endpoint will appear on the Endpoints section**
+   
+![CDNEndpoints](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/q4grlqb887rtky5i6e9l.png)
+
+
+**Important**
+    
+    For Azure CDN from Akamai profiles, you are allowed to change the optimization of a CDN endpoint after it has been created.
+
+    For Azure CDN from Verizon profiles, you cannot change the optimization of a CDN endpoint after it has been created.
+
+4. **After some minutes, navigate to CDN endpoint**
+
+```
+cdn-endpoint-app-green-dev-001.azureedge.net
+```
+
+You see the same page that you ran earlier in an Azure web app. Azure CDN has retrieved the origin web app's assets and is serving them from the CDN endpoint.
+
+To ensure that this page is cached in the CDN, refresh the page. Two requests for the same asset are sometimes required for the CDN to cache the requested content.
+ 
+- For Azure CDN Standard from `Microsoft profiles`, propagation usually completes in ten minutes. 
+- For Azure CDN Standard from `Akamai profiles`, propagation usually completes within one minute. 
+- For Azure CDN Standard from `Verizon and Azure CDN Premium` from Verizon profiles, propagation usually completes within 90 minutes.
```

Comment:
est-ce que les temps indiqués est le meme que pour le titre `4. **After some minutes, navigate to CDN endpoint**` ?

si oui, je pense que ça pourrait etre bien d'essayer de les remonter, par exemple, tu pourrais faire avant le `4.` un `Wait for deployment blablabla` avec les explications de qu'est-ce qui se passe en arrière-plan et les délais normaux